### PR TITLE
Release 1.2.0

### PR DESCRIPTION
**Pull request by MikaelSmith**

### CODE REVIEW

**Comment by MikaelSmith** [comment]
Thanks @dhollinger

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,26 +4,31 @@ All notable changes to this project will be documented in this file.
...
+## [1.1.1](https://github.com/voxpupuli/puppetdb-ruby/tree/1.1.1) (2017-08-17)
+[Full Changelog](https://github.com/voxpupuli/puppetdb-ruby/compare/1.1.0...1.1.1)
 
 **Fixed bugs:**
 
 - Rubygem is broken for 1.1.0 [\#31](https://github.com/voxpupuli/puppetdb-ruby/issues/31)
 
 **Merged pull requests:**
 
+- Release 1.1.1 [\#32](https://github.com/voxpupuli/puppetdb-ruby/pull/32) ([dhollinger](https://github.com/dhollinger))
```
Comment (ekohl) [inline CHANGELOG.md]: In other projects we usually skip these by adding the `skip-changelog` label.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,26 +4,31 @@ All notable changes to this project will be documented in this file.
...
 - Add badges [\#30](https://github.com/voxpupuli/puppetdb-ruby/pull/30) ([raphink](https://github.com/raphink))
+- Release 1.1.0 [\#29](https://github.com/voxpupuli/puppetdb-ruby/pull/29) ([dhollinger](https://github.com/dhollinger))
```
Comment (ekohl) [inline CHANGELOG.md]: Same here

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,26 +4,31 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not impact the functionality of the module.
 
-## Unreleased
+## [v1.2.0](https://github.com/voxpupuli/puppetdb-ruby/tree/v1.2.0) (2019-08-06)
```
Comment (ekohl) [inline CHANGELOG.md]: All releases are not prefixed with a `v` so personally I'd continue to be consistent. Not sure what other maintainers think of this.

Comment (MikaelSmith) [reply CHANGELOG.md]: Ok. This seems to be a thing the rake task is adding.

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,18 +4,21 @@ All notable changes to this project will be documented in this file.
...
 
-- Add AccessDeniedError, UnauthorizedError and ForbiddenError exceptions
-- Silence warnings about unspecified exception types in spec tests
-- Add support for Puppet Enterprise RBAC token authentication
-- Add support for loading configuration defaults from Puppet Enterprise Client Tools configuration files
-  <https://puppet.com/docs/pe/2018.1/installing_pe_client_tools.html#configuring-and-using-client-tools>
+- Add PE RBAC token support [\#34](https://github.com/voxpupuli/puppetdb-ruby/pull/34) ([seanmil](https://github.com/seanmil))
 
-## [v1.1.1](https://github.com/voxpupuli/puppetdb-ruby/tree/v1.1.1) (2017-08-17)
-[Full Changelog](https://github.com/voxpupuli/puppetdb-ruby/compare/1.1.0...v1.1.1)
```
Comment (MikaelSmith) [inline CHANGELOG.md]: Since the actual tag was `1.1.1`, these links were previously incorrect.

**Comment by MikaelSmith** [comment]
Made the requested fixes to the changelog. Still manually excluding previous release PRs, they need the `skip-changelog` label applied.

File: CHANGELOG.md (hunk 3 above)
Comment (ekohl) [reply CHANGELOG.md]: This is caused by:
https://github.com/voxpupuli/puppetdb-ruby/blob/ce306366231f7a805a535a857f38e97288bbcea3/Rakefile#L7

Comment (MikaelSmith) [reply CHANGELOG.md]: Yes, I removed the `v`.

**Pull request merged**